### DISCUSSION TITLE
Windows10, CMake has been installed but it showed the following error while I'm executing setup.cmd in the zephyr-sdk-0.14.1 folder.

### DISCUSSION THREAD

**Discussion by kp008114** [zephyrproject-rtos/zephyr#45402] [Q&A]
Windows10, CMake has been installed but it showed the following error while I'm executing setup.cmd in the zephyr-sdk-0.14.1 folder. I also added CMake into PATH of environment variable.
![camke_path_exist](https://user-images.githubusercontent.com/104830103/167064752-2e053591-f020-4af4-837b-604980f70deb.jpg)
![cmake_prompt_error](https://user-images.githubusercontent.com/104830103/167064756-0663564c-fe0d-40ba-b313-d3cf531292db.jpg)

**Comment by AhmedHaisaraQuasar** [comment]
Hello, I have the same issue and want to know if you succedeed to get a solution ?

**Comment by glenn-andrews** [comment]
I had a student with the same problem. Somehow `C:\Windows\System32` was not in their path. When they added it `setup.cmd` worked.
